src/useradd.c

```diff
@@ -1280,6 +1282,7 @@ static void process_flags (int argc, char **argv)
 			{"user-group",     no_argument,       NULL, 'U'},
 #ifdef WITH_SELINUX
 			{"selinux-user",   required_argument, NULL, 'Z'},
+			{"selinux-range",  required_argument, NULL, 202},
```

Comment:
As a nit pick, I'm not sure if all the long-only options are counted across all binaries, but for me it would make sense to have the same ID for both `usermod` and `useradd`. But again, just a nitpick.